src/main/scala/mimir/statistics/StatsPlugins.scala

```diff
@@ -0,0 +1,118 @@
+package mimir.statistics
+
+import mimir.Database
+import mimir.algebra._
+
+/**
+ * StatsPlugins - defines various types of statistical anaysis methods like:
+ *  RSD (Relative Standard Deviation) - standardized measure of dispersion of a distribution.
+ *  R-Squared - statistical measure of how close the data are to the fitted regression line. 
+ *  score - find the statistical score evaluation for a column based on any of the above said methods.
+ *  combinedScore -  find the statistical score evaluation for a column using weighted average of RSD and R-Squared score.
+ */
+object StatsPlugins{
```

Comment:
"StatsUtils" is perhaps a name for this object that is more consistent with naming conventions used elsewhere in mimir.   I will change it.